packages/core/lib/commands/test.js

```diff
@@ -67,81 +63,27 @@ const command = {
       Environment.detect(config).catch(done);
     }
 
-    let ipcDisconnect;
-    let files = [];
+    let ipcDisconnect, files;
 
-    if (options.file) {
-      files = [options.file];
-    } else if (options._.length > 0) {
-      Array.prototype.push.apply(files, options._);
-    }
-
-    try {
-      if (files.length === 0) {
-        const directoryContents = glob.sync(
-          `${config.test_directory}${path.sep}*`
-        );
-        files =
-          directoryContents.filter(item => fs.statSync(item).isFile()) || [];
-      }
-    } catch (error) {
-      return done(error);
-    }
-
-    files = files.filter(function(file) {
-      return file.match(config.test_file_extension_regexp) != null;
-    });
-
-    let temporaryDirectory;
     try {
-      temporaryDirectory = temp.mkdirSync("test-");
+      files = determineTestFilesToRun(options, config);
```

Comment:
Think it's safe to use just the `config` here, not the `options`? We've already `detect`ed against the options.

I can't decide what I think is better; what do you think?